include/zephyr/drivers/flash.h

```diff
@@ -471,6 +471,16 @@ __syscall int flash_ex_op(const struct device *dev, uint16_t code,
 #define FLASH_EX_OP_VENDOR_BASE 0x8000
 #define FLASH_EX_OP_IS_VENDOR(c) ((c) & FLASH_EX_OP_VENDOR_BASE)
 
+/**
+ *  @brief Enumeration for extra flash operations
+ */
+enum flash_ex_op_types {
+	/*
+	 * Sends Flash reset SPI commands to reset the flash device.
+	 */
+	FLASH_RESET = 0,
```

Comment:
FLASH_EX_OP_RESET, keep the prefix for ex ops the same as for the existing define for base.